internal/portforward/loop.go

```diff
@@ -157,6 +157,17 @@ func (l *Loop) GetPortsForwarded() (ports []uint16) {
 	return l.service.GetPortsForwarded()
 }
 
+func (l *Loop) SetPortsForwarded(ports []uint16) {
+	if l.service == nil {
+		return
+	}
+
+	err := l.service.SetPortsForwarded(l.runCtx, ports)
+	if err != nil {
+		l.logger.Error(err.Error())
+	}
```

Comment:
Commit f18cdb8a56b1cdb0bc9cd9644a2aaa744c83333f addresses this. It  would probably suffice to let control server respond with a more generic error, since the original error message already gets logged anyways. What do you think?

Reply:
I suggested exactly that 😄 I should had read unresolved conversations better!